Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -316,7 +311,7 @@ def main():
             return_outputs(*Druva_UpdateQuarantineRange_Command(clientObj,
                                                                 resource_id, resource_type, range_id, from_date,
                                                                 to_date))
-            # return_outputs(*Druva_ListQuarantineRanges_Command(clientObj))
+            return_outputs(*Druva_ListQuarantineRanges_Command(clientObj))
```

Comment:
same here

Reply:
This was done to return the output as requested in the previous review.

Reply:
I meant, if you have the second why do you need the first?